python_cdk/src/detectors/python-cdk-open-search-encrypted-at-rest/python-cdk-open-search-encrypted-at-rest_compliant.py

```diff
@@ -0,0 +1,16 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+# {fact rule=python-cdk-open-search-encrypted-at-rest@v1.0 defects=0}
+import aws_cdk as cdk
+from aws_cdk import Stack
+from aws_cdk.aws_opensearchservice import EncryptionAtRestOptionsProperty
+from aws_cdk.aws_elasticsearch import CfnDomain as LegacyCfnDomain
+
+class CdkStarterStack(cdk.Stack):
+    def __init__(self, scope: cdk.App, id: str):
+      super(scope, id)
+
+      # Compliant: `encryption_at_rest_options` is set.
```

Comment:
"# Compliant: The LegacyCfnDomain instantiation sets the encryption_at_rest_options property with EncryptionAtRestOptionsProperty, enabling encryption at rest."